tests/python-pytest/onnx/test_onnxruntime.py

```diff
@@ -815,3 +815,66 @@ def test_dynamic_shape_bert_inference_onnxruntime(tmp_path, model):
 
     finally:
         shutil.rmtree(tmp_path)
+
+
+@with_seed()
+@pytest.mark.parametrize('model_name', [('awd_lstm_lm_600', 600), ('awd_lstm_lm_1150', 1150)])
+@pytest.mark.parametrize('seq_length', [16, 32])
```

Comment:
Shall we also test other seq_length like 128 or 256?

Reply:
Yeah I can change them to 96/128, which are two commonly used lengths